src/main/java/com/cvsgo/dto/review/ReadReviewResponseDto.java

```diff
@@ -0,0 +1,82 @@
+package com.cvsgo.dto.review;
+
+import com.cvsgo.entity.ReviewImage;
+import com.cvsgo.entity.User;
+import com.cvsgo.entity.UserTag;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class ReadReviewResponseDto {
+
+    private final Long reviewId;
+
+    private final Long reviewerId;
+
+    private final String reviewerNickname;
+
+    private final String reviewerProfileImageUrl;
+
+    private final Boolean isFollowingUser;
+
+    private final Boolean isMe;
+
```

Comment:
```suggestion

    private final Boolean isReviewLiked;
```
사용자의 리뷰 좋아요 여부가 빠진거같슴다